src/20_Components/amp-social-share.html

```diff
@@ -21,48 +21,68 @@
+    <amp-social-share type="gplus"></amp-social-share>
+    <amp-social-share type="email"></amp-social-share>
+    <amp-social-share type="pinterest"></amp-social-share>
+    <amp-social-share type="linkedin"></amp-social-share>
+    <amp-social-share type="facebook" data-param-app_id="254325784911610"></amp-social-share>
   </div>
 
   <!-- #### Configuration -->
   <!--
     Embed the `amp-social-share` widget [choosing a type](https://www.ampproject.org/docs/reference/components/amp-social-share#types), then configure the actions.
 
-    * `text` is the text to include in the share
-    * `url` is the URL to share
-    * `attribution` is where the share is attributed to.
+    * `width`, default 60px.
+    * `height`, default 44px.
+    * `data-param-text` is the text to include in the share.
+    * `data-param-url` is the URL to share, current URL by default.
+    * `data-param-attribution` is where the share is attributed to.
+
+
+    All `data-param-*` prefixed attributes will be turned into URL parameters and passed to the share endpoint.
   -->
-  <amp-social-share type="linkedin" width="60" height="44"
-    data-text="Check out these AMP Examples!"
-    data-url="https://ampbyexample.com/"
-    data-attribution="AMPhtml">
+  <amp-social-share type="linkedin"
+    width="40"
+    height="40"
+    data-param-text="Check out these AMP Examples!"
+    data-param-url="https://ampbyexample.com/"
+    data-param-attribution="AMPhtml">
   </amp-social-share>
 
-  <!-- #### Customized Views -->
+  <!-- #### Custom Style -->
   <!--
-    Sometimes you want to provide your own style. In this instance, you can **embed an anchor without a `href`** and it will be populated. This provides you the flexibility to build your own UI for the share button.
+    Whenever you want to provide your own style, use CSS properties within the `<style amp-custom>` element ([Modify the presentation](https://www.ampproject.org/docs/get_started/create/presentation_layout)). This will make sure the image stays responsive and centered, and the default styling is overwritten.
+
+    * `background-color`, if you want the element to be a different color.
+    * `background-image`, provide another image if you want to change the icon.
+    * `background-repeat` (when using `background-image`), set to `no-repeat`.
+    * `background-position` (when using `background-image`), set to `center`.
+    * `background-size` (when using `background-image`), set to `contain`.
   -->
-  <amp-social-share type="linkedin" width="60" height="44" data-text="The AMP Project" data-url="https://www.ampproject.org/" data-attribution="amphtml">
-    <a id="customized-social-share" class="custom">
-      <amp-img src="https://raw.githubusercontent.com/google/material-design-icons/master/social/1x_web/ic_share_white_48dp.png" width="18" height="18" layout="responsive" alt="an image">
-      </amp-img>
-    </a>
+  <amp-social-share type="linkedin"
+    width="76"
+    height="22"
```

Comment:
No problem, but maybe I should remove the height+width completely? The point of that example is to show the custom styling, which doesn't interfere with the default behavior. I used those dimensions to show that the image scales and stays within the container, default is a square icon within a rectangle shape as well. I agree the buttons look better when they are square, but I don't think that should be the point of that example. Making smaller examples that people can quickly understand and pick and choose to their desired result should be the focus. So I'd vote for just the type and class attributes for this example.